README.md

```diff
@@ -1,5 +1,49 @@
-# injected
-Source code of DLLs / assemblies for injection into target process
+# injectdll
 
-usage:
-py -2.7 example_injector.py \<Process name\> \<path to dll\>
+injectdll is a Python package that allows you to inject DLLs into running processes on Windows (32 or 64-bit).
+
+This can be useful for various tasks, such as debugging or adding functionality to existing applications.
+
+It contains test DLLs for classic Win32 API and WPF applications.
+
+Link to PyPi: <https://pypi.org/project/injectdll/>
+
+## How to use
+
+    python.exe example_injector.py \<Process name\> \<path to dll\> 
+
+## Development
+
+ 1. Clone repository: <https://github.com/pywinauto/injected.git>
+ 2. Create new features or improve exiting (for the injectdll package)
+ 3. After work run setup.py from project root to check standalone
+
+        python.exe setup.py bdist_wheel
+
+## Requirements
+
+### For use as package
+
+  * Python >= 3.7
+  * pywin32 >= 306
+  * six >= 1.16.0
+
+### For development purpose
+
+  * setuptools >= 65.5.1
+  * wheel >= 0.38.4
+  * .NET Framework Targeting Pack >= 4.8
+  * .NET Compiler Platform
+  * MSBuild
+  * MSVC >= 143
+  * cmake >= 3.26.3
+
+  **NOTE**: You may use PyCharm + Visual Studio with C++/C# components for development
```

Comment:
First the Python requirements are very typically already placed in `requirements.txt` (for use) and `dev-requirements.txt` (for development). Second the build and test instruction is better to move to `CONTRIBUTING.md`. So it makes sense to describe only development requirements for C/C++/C# parts and a reference to `dev-requirements.txt` for Python parts.